usage-examples/csharp/driver/Examples/Aggregation/GroupTotal.cs

```diff
@@ -0,0 +1,134 @@
+//	:replace-start: {
+//	  "terms": {
+//	    "_order": "order",
+//      "_aggDb": "aggDb"
+//	  }
+//	}
+namespace Examples.Aggregation;
+
+using MongoDB.Driver;
+using MongoDB.Bson;
+
+public class GroupTotal
+{
+    private IMongoDatabase _aggDB;
+    private IMongoCollection<Order> _orders;
+    
+    public void SeedData()
+    {
+        DotNetEnv.Env.TraversePath().Load();
+        string uri = DotNetEnv.Env.GetString("CONNECTION_STRING", "Env variable not found. Verify you have a .env file with a valid connection string.");
+        // :snippet-start: connection-string
+        // :uncomment-start:
+        //var uri = "mongodb+srv://mongodb-example:27017";
+        // :uncomment-end:
+        // :snippet-end:
+        var client = new MongoClient(uri);
+        _aggDB = client.GetDatabase("agg_tutorials_db");
```

Comment:
Idiomatic C# doesn't use `_variable` except in private getters/setters. Also, it's recommended that variables names are more explicit. In this case, I'd go with `sampleDatabase` or `aggregationDatabase` or similar.

Reply:
same as all cases below this, too.